clients/mobile/app.json

```diff
@@ -12,9 +12,35 @@
       "backgroundColor": "#ffffff"
     },
     "ios": {
-      "supportsTablet": true
+      "supportsTablet": true,
+      "bundleIdentifier": "com.example.mobile",
```

Comment:
we have a domain name, that we can use

Reply:
Updated the `bundleIdentifier` to `net.bloodconnect.app` to match our domain `bloodconnect.net` instead of `com.example.mobile`.